storybook/src/elements/nuxeo-video/nuxeo-video-info.stories.js

```diff
@@ -0,0 +1,26 @@
+import { html } from 'lit-html';
+import { storiesOf } from '@storybook/polymer';
+import '@nuxeo/nuxeo-ui-elements/nuxeo-video/nuxeo-video-info';
+import DocumentBuilder from '../../data/documents.data';
+import videoProperties from '../../data/video.data.js';
+
+storiesOf('UI/nuxeo-video', module).add('Nuxeo Video Info', () => {
+  const documentBuilder = new DocumentBuilder()
+    .setType('File')
+    .setProperties({ 'vid:info': videoProperties['vid:info'] });
```

Comment:
This story is broken because `videoProperties['vid:info']` is undefined.
Looking to `video.data.js` it needs to be something like `videoProperties['vid:transcodedVideos'][0].info`, right?

Reply:
actually no, it's missing because of a merge conflict in the rebase process. i added it back and is working fine